Tests/TestingApps/SimpleApp/SimpleApp/SimpleApp.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SimpleApp.swift
+//  SimpleApp
+//
+//  Created by Nacho Soto on 5/30/23.
+//
+
+import RevenueCat
+import SwiftUI
+
+#warning("This needs to be set")
+private let apiKey = ""
+private let proxyURL = ""
```

Comment:
let's add another warning here that explains that this one is optional and you can set it to your own instance of the RC backend or leave empty for prod